bigquery_etl/cli/monitoring.py

```diff
@@ -156,6 +157,14 @@ def update(name: str, sql_dir: Optional[str], project_id: Optional[str]) -> None
                                     metric.metric_type.predefined_metric
                                 )
 
+                        if (
+                            metadata.monitoring.collection
+                            and collection.collection is None
+                        ):
+                            collection.collection = SimpleCollection(
+                                name=metadata.monitoring.collection
+                            )
```

Comment:
This doesn't seem to use the `deployment` values, so I don't think it needs to be done in that loop, but could be done in the parent `collection` loop.
```suggestion
                    if (
                        metadata.monitoring.collection
                        and collection.collection is None
                    ):
                        collection.collection = SimpleCollection(
                            name=metadata.monitoring.collection
                        )
```